src/ILLink.RoslynAnalyzer/DataFlow/LocalDataFlowVisitor.cs

```diff
@@ -111,12 +112,12 @@ public override TValue VisitSimpleAssignment (ISimpleAssignmentOperation operati
 				// Doesn't get called for assignments to locals, which are handled above.
 				HandleAssignment (value, targetValue, operation);
 				break;
-			case IPropertyReferenceOperation propertyRef:
-				// A property assignment is really a call to the property setter.
-				var setMethod = propertyRef.Property.SetMethod;
-				Debug.Assert (setMethod != null);
-				HandlePropertySetterArgument (value, setMethod!, operation);
-				break;
+			case IPropertyReferenceOperation propertyRef: {
```

Comment:
Nit: I don't think these braces are providing any value

Reply:
True - I'll remove it.